recipes/gwollum/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "gwollum" %}
+{% set version = "2.3.12" %}
+{% set versions = version.split('.') %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: "http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.xz"
+   sha256: 4416c3ccd18e37f9de00fc29a0dff0c25fc9fd511e542611b25c642fac882edc
+   patches:
+     # use C++17 standard (required to link against ROOT>=6.14)
+     - c++17.patch
+     # link only against libgsl, not libgslcblas
+     - gsl-library.patch
+
+build:
+  error_overdepending: true
+  error_overlinking: true
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage("gwollum", max_pin="x.x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.9.0
+    - pkg-config  # [unix]
```

I think you need make:
```suggestion
    - pkg-config  # [unix]
    - make  # [unix]
```